launcher/src/components/UI/base-header/components/modals/UpdatePanel.vue

```diff
@@ -366,11 +366,11 @@ const searchOsUpdates = async () => {
 
 const getUpdatablePackagesCount = async () => {
   try {
-    const packagesCount = await ControlService.getCountOfUpdatableOSUpdate();
-    const numPackages = Number(packagesCount);
-    numberOfUpdatablePackages.value = isNaN(numPackages) || !numPackages ? 0 : numPackages;
+    const packages = await ControlService.getUpgradeablePackages();
```

Comment:
if you're using `getUpgradeablePackages` you can directly save the statue in the pinia store so the ui doesn't need to load the data again: https://github.com/stereum-dev/ethereum-node/blob/main/launcher/src/components/UI/server-management/components/server-update/UpdateServer.vue#L164-L176

Reply:
fixed